src/components/ModeIcon.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useMemo } from 'react'
+
+import Avatar, { nameToColor } from 'cozy-ui/transpiled/react/Avatar'
+
+import { pickModeIcon } from 'src/components/helpers'
+
+const ModeIcon = ({ mode }) => {
+  const color = useMemo(() => nameToColor(mode), [mode])
+
+  return (
+    <Avatar
+      style={{ backgroundColor: color }}
```

Comment:
nit: We should never create a style object like this. If we have a parent’s rerender we will create a new style object and then we’ll rerender avatar. 

You can create the style object outside of the component if possible. Or you can useMemo the object

Reply:
obsolète corrigé plus loin, mais oui tout à fait !